app/jobs/series_update_job.rb

```diff
@@ -0,0 +1,49 @@
+require 'prx_access'
+
+class SeriesUpdateJob < ActiveJob::Base
+  include Announce::Subscriber
+  include PRXAccess
+
+  queue_as :feeder_default
+
+  subscribe_to :series, [:create, :update, :delete]
+
+  attr_accessor :body, :podcast, :series
+
+  def perform(*args)
+    ActiveRecord::Base.connection_pool.with_connection do
+      super
+    end
+  end
+
+  def receive_series_update(data)
+    load_resources(data)
+    if podcast
```

Comment:
And I'm assuming that this will actually pick up create messages ... which _should_ be largely a no-op, since CMS just POSTed very similar data.